public/app/features/logs/components/panel/LogLine.tsx

```diff
@@ -140,6 +174,43 @@ export const getStyles = (theme: GrafanaTheme2) => {
           width: '100%',
         },
       },
+      '& .log-syntax-highlight': {
+        '.token.log-token-timestamp': {
```

Comment:
I don't love all the specificity in this CSS, but if we need it at least can we &.token {} so the .token isn't needed in each selector?

Reply:
Definitely. The .token wasn't really necessary.